tensorflow_addons/image/transform_ops.py

```diff
@@ -37,6 +40,7 @@ def transform(
     transforms: TensorLike,
     interpolation: str = "NEAREST",
     fill_mode: str = "CONSTANT",
```

Comment:
Could we set these set these strings to small caps `"nearest"` and `"constant"` (in the `Args` below as well)? This would be more consistent with the docstrings in `tf.keras.layers` and we have an `.upper()` in the code anyways.

Reply:
Updated.